From 58f870d005d48f299785e7dbd0f153dd544f2a5a Mon Sep 17 00:00:00 2001
From: Ben Pfaff <blp@nicira.com>
Date: Mon, 2 May 2011 17:15:21 -0700
Subject: [PATCH] docs: Suppress "warning: macro `DD' not defined" warnings
 from man.

deamon.man allows the file that is including it to include extra text in
the description of --detach by defining a macro named DD.  Only some of the
manpages that included it did this (only those manpages that needed extra
text there).  But it's better to be quiet in "man --warnings", so this
defines DD to an empty value in the other manpages that include daemon.man.

Reported by lintian.
---
 ovsdb/ovsdb-client.1.in       |    1 +
 utilities/ovs-controller.8.in |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/ovsdb/ovsdb-client.1.in b/ovsdb/ovsdb-client.1.in
index fe7b7fa..7d8d0b0 100644
--- a/ovsdb/ovsdb-client.1.in
+++ b/ovsdb/ovsdb-client.1.in
@@ -133,6 +133,7 @@ The following options controlling output formatting:
 .SS "Daemon Options"
 The daemon options apply only to the \fBmonitor\fR command.  With any
 other command, they have no effect.
+.ds DD
 .so lib/daemon.man
 .SS "Logging Options"
 .so lib/vlog.man
diff --git a/utilities/ovs-controller.8.in b/utilities/ovs-controller.8.in
index aa5751f..18d77de 100644
--- a/utilities/ovs-controller.8.in
+++ b/utilities/ovs-controller.8.in
@@ -133,6 +133,7 @@ Use this option more than once to add flows from multiple files.
 .SS "Public Key Infrastructure Options"
 .so lib/ssl.man
 .so lib/ssl-peer-ca-cert.man
+.ds DD
 .so lib/daemon.man
 .so lib/vlog.man
 .so lib/unixctl.man
-- 
1.7.4.4

